openide.dialogs/manifest.mf

```diff
@@ -1,6 +1,6 @@
 Manifest-Version: 1.0
 OpenIDE-Module: org.openide.dialogs
-OpenIDE-Module-Specification-Version: 7.43
+OpenIDE-Module-Specification-Version: 7.44
```

Comment:
Notice: I continue to vote against `Leafable` to become part of the API, but...

...should the `Leafable` become part of the API, then all modules that use the interface shall increase the dependency on `org.openide.dialogs` to `7.44`. That would give enough information to the NetBeans Runtime container to prevent such modules to run on NetBeans 8.2.

Reply:
I also vote against `Leafable` to become part of the API. From my point of view a presenter being "Leafable" is an implementation detail, and should not become part of a public API.

Reply:
True, but what if the user can't use a `DialogDisplayer` for some reason? How would it know that a window shouldn't be a parent?

Reply:
I would like to remind you that this wouldn't be an issue if the Mac considered application modal windows to be application modal and keep them on top of all application windows.